spec/linter-stylelint-spec.js

```diff
@@ -7,117 +7,136 @@ describe('The stylelint provider for Linter', () => {
 
   beforeEach(() => {
     atom.workspace.destroyActivePaneItem();
+    atom.config.set('linter-stylelint.usePreset', true);
+
     waitsForPromise(() => {
-      atom.packages.activatePackage('linter-stylelint');
```

Comment:
These actually need to execute sequentially, from my understanding `Promise.all()` simply fires all off (sequentially, but not waiting between), and waits for them all to resolve.

For this specific plugin this probably doesn't make a difference, but once `activationHooks` are fixed in v1.4.0 I plan to move all plugins to only activating when a language they can handle is loaded, in which case the current method was the only one we found to work currently. (linter-php is the only plugin currently using `activationHooks`)